force-app/main/default/classes/CSUtils.cls

```diff
@@ -382,8 +384,26 @@ public with sharing class CSUtils {
 		email.setSubject(subject);
 		email.setPlainTextBody(message);
 		email.setHtmlBody(message);
+		if(canSendEmails()){
+			Messaging.sendEmail(new Messaging.SingleEmailMessage[] {email});
+		}
+	}
+
+	private static Boolean orgCanSendEmails = null;
```

Comment:
Add a method to check if we can send an email so tests don't fail if sandbox email permission is disabled.